### ISSUE TITLE
ATM Hacking Payout and animation

### ISSUE THREAD

**Issue by MrE1ectric** [echoroleplay/feedback#143]
**Suggestion Description**
Have the ATM pay up to $10,000 in Marked bills and/or Lose notes. Also if we have to take the money from the atm like when you do in a bank for the table to add some time when the hack is done as it can be done in less than a min now and doesn't give time for cops to show

**Reason**
Payout:
The payout is averaging anywhere from 5-7 notes from all the people I have spoken to and also done myself. The average price for the loose note is $60 per meaning that we are getting anywhere from $300-$420 when gangs are selling the device for $2,500-$5,000 then if you get caught it's not even worth the arrest due to the fines and time.

VS

When you hit a bank and the money gets out but somebody is caught they get a $5,000-$7,000 fine and time but the payout makes it worth it as they can cover the cost easily 

Animation:
The hack is super quick and if cops respond this allows a bit more interaction with it other than okay I have the money let's go before they show up.

**Additional context**
I am adding this suggestion to add some value in ATM hacking and a bit more RP behind it as well.

**Comment by damightylamb** [comment]
This is a wild suggestion asking for $10,000 lmaoooo

**Comment by MrE1ectric** [comment]
I mean i could ask for $200,000 lol but it can be random and based on where it is will change the payout and how long you have to stay there for the money to be released.

**Comment by damightylamb** [comment]
It's a petty crime and you're gonna get petty crime payouts

**Comment by damightylamb** [comment]
It's a petty crime and you're gonna get petty crime payouts

**Comment by SurgeGit** [comment]
Yeah agreed. It should be more than robbing a store which takes 10 seconds. ATMs obviously carry more money than a store and you have to hack it as well.

**Comment by XxThaHitmanxX** [comment]
I think 2 to 3 marked bills would be a realistic payout for an atm would make sense. As people have said atms do normally carry more money than stores do and store pay straight cash, at least atm pays in illegal currency so is not a quick payout. Could even think about rolls. I have received up to 7 notes on an atm so 7 rolls which is roughly 5 to 6k is more realistic. Essentially you just take the cash in the atm and an rolled it up for the easy convenience of taking the money so that would also work.

**Comment by SuspiciouslyStickySock** [comment]
You can complete the hack stupidly fast, yeah it should pay more than a store but not by much
